src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -68,6 +68,12 @@
 #define UCT_IB_MLX5_ATOMIC_MODE          3
 #define UCT_IB_MLX5_CQE_FLAG_L3_IN_DATA  UCS_BIT(28) /* GRH/IP in the receive buffer */
 #define UCT_IB_MLX5_CQE_FLAG_L3_IN_CQE   UCS_BIT(29) /* GRH/IP in the CQE */
+#define UCT_IB_MLX5_CQE_COMPRESSED       0x3
+#define UCT_IB_MLX5_CQE_FORMAT_MASK      0xc
+#define UCT_IB_MLX5_CQE_FORMAT(_op_own) \
+    (((_op_own)&UCT_IB_MLX5_CQE_FORMAT_MASK) >> 2)
+#define UCT_IB_MLX5_CQE_FORMAT_CLEAR(_op_own) \
+    (_op_own &= ~UCT_IB_MLX5_CQE_FORMAT_MASK)
```

Comment:
this macro should not be; only hw writes the cqe. sw should not overwrite it.
and in decompressed cqe, we should not care to set op_own

Reply:
Discussed verbally, please reopen in case of questions.